.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+yarnPath: .yarn/releases/yarn-sources.cjs
```

Comment:
What is yarn-sources.cjs?

Reply:
It appears to be some webpack-packed code compressed and base64’d.

Reply:
Is this a pack of yarn@2’s source? How was the pack generated? The process for updating this file should be covered in README.md if it isn’t handled automatically by Yarn itself.

Reply:
Will update the readme. This file is the result of running `yarn set version from sources`, which builds Yarn from the master branch (so as to benefit from the fix we made in https://github.com/yarnpkg/berry/pull/1588). Once the build will have reached stable (in a few weeks), upgrades will be `yarn set version latest`.